src/bin/cargo-flamegraph.rs

```diff
@@ -283,8 +287,20 @@ fn main() {
         .take()
         .unwrap_or("flamegraph.svg".into());
 
+    let should_open = opt.open;
```

Comment:
I think it is cleaner to just use `opt.open` in the if condition below instead of assigning it to a variable.